### PR TITLE
Update packaging dep to v20.8

### PR DESCRIPTION
**Pull request by gi0baro**
Resolves: python-poetry/poetry#3458

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

**Comment by damienrj** [comment]
This should also resolve issue https://github.com/python-poetry/poetry/issues/3591 now that I understand the problem better.

**Comment by alexifm** [comment]
Based on the testing over at https://github.com/python-poetry/poetry/pull/3530, this PR does the trick.

**Comment by alexifm** [comment]
@finswimmer Could you review this?

File: vendors/pyproject.toml [hunk 1]
```diff
@@ -23,5 +23,6 @@ python = "^3.6"
 
 jsonschema = "^3.2.0"
 lark-parser = "^0.9.0"
-packaging = "^20.1"
+packaging = "^20.8"
+pyrsistent = "^0.16.0"
```
Comment (sdispater) [inline vendors/pyproject.toml:27]: Why is `pyrsistent` added explicitly here?

Comment (gi0baro) [reply vendors/pyproject.toml:27]: @sdispater 'cause in ^0.17 they dropped Python 2.x support (ref: https://github.com/tobgu/pyrsistent/blob/master/CHANGES.txt#L17#L19)

Comment (sdispater) [reply vendors/pyproject.toml:27]: @gi0baro Thanks for the information 👍

**Review by sdispater — APPROVED** [review]

**Comment by henryiii** [comment]
Hoping that Packaging 20.9 will be released soon (maybe today!) and that will also add support for Universal wheels on Apple Silicon, so it would be a fantastic upgrade (okay, not quite as important yet as loading any wheel on Intel Big Sur, which was the core of this update, but still and important step forward too!). https://github.com/pypa/packaging/issues/385

**Comment by alexifm** [comment]
Thanks @sdispater!

**Comment by henryiii** [comment]
Packaging 20.9 is out, can we get an update? It's needed for proper Apple Silicon support.

**Comment by damienrj** [comment]
Any updates on a release? I had just switched my team to poetry before this started and hasn't been a great experience for everyone so we are looking forward to the release.

**Comment by henryiii** [comment]
I'd love to see an update too; not being able to install NumPy on macOS Big Sur is a quite a bummer! I asked for Packaging 20.9 largely so it would be possible to be included in the Poetry 1.1.x series, since 21.0 will drop Python 2.7 and 3.5 (perfect for Poetry 1.2).

**Comment by matteosantama** [comment]
Looking forward to a release as well, as I'm sure are lots of other Big Sure users out there :)

**Comment by JonasKlasen** [comment]
Hi @sdispater I would really appreciate if this could be released or are there any reasons not to do so?

**Comment by damienrj** [comment]
Has this fixed the problem for others? I still can't install packages
https://github.com/python-poetry/poetry/issues/3591#issuecomment-778862630

**Pull request merged**